sig-and-wg/sig-core/proposals/installer-release.md

```diff
@@ -0,0 +1,29 @@
+# Kyma Installer with embedded kyma charts and tiller
+
+Created on 2018-09-19 by Piotr Bochyński (@pbochynski).
+
+## Status
+
+Proposed on 2018-09-19
+
+## Abstract
+
+Current installation process involves downloading kyma charts from cloud storage. Such solution allows to release kyma installer and kyma charts independently. In the consequence, there is possibility to install different versions (or flavors) of kyma components using single installer binary. It seems to be a nice feature, but in the reality introduces problems:
```

Comment:
`Kyma` -> capitalize x4
`the cloud storage` -> add article
`the Kyma installer` - add article
~In the~ consequence -> `As a consequence`
`a single installer binary` -> add article

`It seems to be a convenient feature but in reality, it introduces the following problems:`
"Nice" is too generic word, such as Polish "fajny". Try to avoid it while describing features. Maybe "convenient" is a good word here?